k-distribution/tests/regression-new/bison-glr-bug/Makefile

```diff
@@ -0,0 +1,17 @@
+DEF=iele
+EXT=
+TESTDIR=.
+KOMPILE_FLAGS+=--gen-glr-bison-parser
+
+test: test.kore
+	cat test.kore | diff - test.ref
+
+test.kore: forwarder.iele kompile
+	./iele-kompiled/parser_PGM forwarder.iele > test.kore
+include ../../../include/kframework/ktest.mak
+
+clean:
+	rm -rf test.kore test-kompiled
```

Comment:
This is missing some file patterns.
I've updated the list in my warnings PR:
`rm -rf $(KOMPILED_DIR) .depend-tmp .depend .krun-* .kprove-* kore-exec.tar.gz`
Is it possible to delete test.kore and still call the command from ktest.mak?

Reply:
You're welcome to suggest how to do that, but I don't really know myself.